src/container-comparison/CompareContainersWidget.tsx

```diff
@@ -122,12 +134,33 @@ const CompareContainersWidgetInner = ({
     );
   }, [parent]);
 
+  const renderAlert = useCallback(() => {
+    // Show this alert if the only change is a local override to a text component
+    if (localUpdateAlertCount > 0) {
+      return (
+        <Alert variant="info">
+          <FormattedMessage
+            {...messages.localChangeInTextAlert}
+            values={{
+              blockName: localUpdateAlertBlockName,
+              count: localUpdateAlertCount,
+              b: BoldText,
+            }}
+          />
+        </Alert>
+      );
+    }
+
+    return null;
+  }, [localUpdateAlertCount, localUpdateAlertBlockName]);
+
   if (isError || isLibError || isContainerTitleError) {
     return <ErrorAlert error={error || libError || containerTitleError} />;
   }
 
   return (
-    <div className="row">
+    <div className="row justify-content-center">
+      {renderAlert()}
```

Comment:
Nit: I think you could just put the `if () {... <Alert> ... </Alert> }` code here directly; I don't really see an advantage to making it a callback function.